Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -1,23 +1,33 @@
 prototype(Neos.NeosIo:Reference.List) < prototype(TYPO3.Neos:Content) {
+	referenceType = ${request.arguments.referenceType}
+	@context.referenceType = ${this.referenceType}
+	searchTerm = ${request.arguments.search}
+	@context.searchTerm = ${this.searchTerm}
+	order = ${request.arguments.order}
+	@context.order = ${this.order}
+	limit = ${request.arguments.limit}
+	@context.limit = ${this.limit}
```

Comment:
Is this the only way to access those filter args from the references collection below?